src/main/java/ratpack/zipkin/internal/DefaultServerTracingHandler.java

```diff
@@ -59,10 +59,19 @@ public void handle(Context ctx) throws Exception {
 
     //place the Span in scope so that downstream code (e.g. Ratpack handlers
     //further on in the chain) can see the Span.
-    Tracer.SpanInScope scope = tracing.tracer().withSpanInScope(span);
+    Tracer.SpanInScope scope = null;
+    try {
+      tracing.tracer().withSpanInScope(span);
+    } catch (Exception e) {
+      if (scope != null) {
+        scope.close();
```

Comment:
If we close this here, the span will no longer be in scope for the rest of the request handling.

Reply:
Ignore these changes